dracut.spec

```diff
@@ -184,6 +184,7 @@ This package contains tools to assemble the local initrd and host configuration.
 Summary: dracut module to build an initramfs with most files in a squashfs image
 Requires: %{name} = %{version}-%{release}
 Requires: squashfs-tools
+Recommends: busybox
```

Comment:
I really don't like this, because otherwise we would `Recommend` any tools we ship for every module we ship, which could explode the installation.

Maybe a dracut subpackage would be a solution?

Reply:
Actually this is already a "dracut-squash" sub package, do you want a dracut-squash-busybox? Or maybe I can just drop this and add a dinfo that busybox is not installed when squash is used.

Reply:
@haraldh we probably end up packaging every module into a sub-package as dracut-module-<module> with proper dependency's etc. since that´s the proper way of doing things.

@ryncsn you could create this as an such sub-package as in dracut-module-squash-busybox

Reply:
@johannbg That will be an empty package, `squash` module itself will detect if busybox is installed and use it if available. How about I just drop this and add a dinfo if busybox is not used?

Reply:
Or I can just drop it silently, it easy to tell if busybox is used for an squashed initramfs with lsinitrd.

Reply:
@ryncsn yeah just drop the recommend, I have already create issue for a sub-packaging scope in which we will be dealing with packaging of the modules so you dont have to worry about that as in we will take care of it.